openml-lightgbm/lightgbm-provider/standard_code_models/extend_model.py

```diff
@@ -0,0 +1,149 @@
+"""
```

Comment:
No, it is a tool that I used to generate >2gb models for the benchmarks. As it might come in handy I don't want to delete it, but maybe I can also place it in the test resources.